archiver.go

```diff
@@ -169,6 +170,12 @@ func updateNodeContent(node *Node, results []saveResult) error {
 	return nil
 }
 
+const chunkerBufSize = 512 * chunker.KiB
+
+var chunkerBufPool = sync.Pool{
```

Comment:
as discussed in IRC, maybe this should live on the chunker type as a non-exported field?